cmd/proxy/actions/health_test.go

```diff
@@ -0,0 +1,30 @@
+package actions
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestHealthHandler(t *testing.T) {
```

Comment:
I am not sure what benefit does this test gives us? It is a unit test that does not check if the system is up or not. I would not add this one